cli/pyproject.toml

```diff
@@ -14,14 +14,11 @@ dependencies = [
 
 [project.optional-dependencies]
 test = [
-    "pytest==8.3.*",
+    "core[test]==0.2.0rc",
     "pytest-mock==3.14.*",
-    "pytest-pixl"
 ]
 dev = [
-    "mypy",
-    "pre-commit",
-    "ruff",
+    "core[dev]==0.2.0rc",
```

Comment:
Do we need the same version as in ["0.2.0rc0"](https://github.com/UCLH-Foundry/PIXL/blob/main/pixl_core/pyproject.toml#L3C12-L3C20)?

Reply:
oops, thanks for spotting that! I've fixed it now